app/src/main/java/ca/bc/gov/bchealth/ui/setting/SettingFragment.kt

```diff
@@ -171,7 +131,7 @@ class SettingFragment : Fragment(R.layout.fragment_setting) {
             positiveBtnMsg = getString(R.string.delete),
             negativeBtnMsg = getString(R.string.cancel),
             positiveBtnCallback = {
-                deleteAllRecordsAndSavedData()
+                bcscAuthViewModel.getEndSessionIntent()
```

Comment:
if the session is already logged out then clicking on delete all records is breaking in the sequence of actions. Please fix this case

Reply:
Also replace the navigation to homeFragment from healthPassFragment